test/e2e_installer/cluster/cluster_test.go

```diff
@@ -155,85 +150,85 @@ var _ = Describe("cluster", func() {
 		Expect(cls.Spec.Machines).Should(HaveLen(1), "Cluster node num is wrong")
 	})
 
-	It("Cluster bootstrap application", func() {
```

Comment:
I perfer to use "/* */" to comments block of code

Reply:
done